common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -326,4 +333,65 @@ public class CommonValue {
             + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"}\n"
             + "    }\n"
             + "}";
+
+    public static final String ML_AGENT_INDEX_MAPPING = "{\n"
+            + "    \"_meta\": {\"schema_version\": "
+            + ML_AGENT_INDEX_SCHEMA_VERSION
+            + "},\n"
+            + "    \"properties\": {\n"
+            + "      \""
+            + MLAgent.AGENT_NAME_FIELD
+            + "\" : {\"type\":\"text\",\"fields\":{\"keyword\":{\"type\":\"keyword\",\"ignore_above\":256}}},\n"
+            + "      \""
+            + MLAgent.AGENT_TYPE_FIELD
+            + "\" : {\"type\":\"keyword\"},\n"
+            + "      \""
+            + MLAgent.DESCRIPTION_FIELD
+            + "\" : {\"type\": \"text\"},\n"
+            + "      \""
+            + MLAgent.LLM_FIELD
+            + "\" : {\"type\": \"flat_object\"},\n"
+            + "      \""
+            + MLAgent.TOOLS_FIELD
+            + "\" : {\"type\": \"flat_object\"},\n"
+            + "      \""
+            + MLAgent.PARAMETERS_FIELD
+            + "\" : {\"type\": \"flat_object\"},\n"
+            + "      \""
+            + MLAgent.MEMORY_FIELD
+            + "\" : {\"type\": \"flat_object\"},\n"
+            + "      \""
+            + MLAgent.CREATED_TIME_FIELD
+            + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
+            + "      \""
+            + MLAgent.LAST_UPDATED_TIME_FIELD
+            + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"}\n"
+            + "    }\n"
+            + "}";
+
+    public static final String ML_MEMORY_META_INDEX_MAPPING = "{\n"
+            + "    \"_meta\": {\"schema_version\": "
+            + ML_MEMORY_META_INDEX_SCHEMA_VERSION
+            + "    },\n"
+            + "    \"properties\": {\n"
+            + "        \"name\" : {\"type\":\"text\",\"fields\":{\"keyword\":{\"type\":\"keyword\",\"ignore_above\":256}}},\n"
+            + "        \"application_type\" : {\"type\":\"keyword\"},\n"
+            + "        \"created_time\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
+            + "        \"last_updated_time\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"}\n"
```

Comment:
Seems can't match the java code https://github.com/opensearch-project/ml-commons/pull/1730/files#diff-c99fc312519dcb3fa952ea768ddcd6139fa9349669c111c186270e2869b98fdf
```
public final static String META_UPDATED_TIME_FIELD = "updated_time";
```
Suggest use the constant just like what you did for MLAgent. That can avoid mismatch.

Reply:
Same suggest for `ML_MEMORY_MESSAGE_INDEX_MAPPING` part

Reply:
makes sense, addressed.